themes/navy/layout/partial/share.swig

```diff
@@ -1,10 +1,9 @@
 <!-- GitHub star -->
-<a class="github-button" href="https://github.com/hexojs/hexo" data-icon="octicon-star" data-show-count="true" aria-label="Star hexojs/hexo on GitHub">Star</a>
+<a href="https://github.com/hexojs/hexo" target="_blank" aria-label="Star hexojs/hexo on GitHub"><img src="https://img.shields.io/github/stars/hexojs/hexo?style=social" alt="Star"></a>
 <!-- GitHub fork -->
-<a class="github-button" href="https://github.com/hexojs/hexo/fork" data-icon="octicon-repo-forked" data-show-count="true" aria-label="Fork hexojs/hexo on GitHub">Fork</a><!-- Tweet button -->
+<a href="https://github.com/hexojs/hexo/fork" target="_blank" aria-label="Fork hexojs/hexo on GitHub"><img src="https://img.shields.io/github/forks/hexojs/hexo?style=social" alt="Fork"></a>
```

Comment:
I think linking to `https://github.com/hexojs/hexo/network/members` is more appropriate since it doesn't require login.

Reply:
https://github.com/hexojs/hexo/network/members only shows people who forks the repo, which is not appropriate.

Reply:
At least it only shows, instead of requesting the visitor to fork. I know the button tries to emulate the fork button of github.com, but I think it shouldn't do that.

I suggest to just link to `https://github.com/hexojs/hexo`, if visitor wants to fork, then it should be done on github.com only. I prefer not to surprise the visitor.

I'm not fond of Members Network link either, since the page takes too long to load.

Reply:
Updated.